_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: default
+title: Create reports with the Reporting CLI
+nav_order: 25
+---
+
+# Create reports with the Reporting CLI
+
+You can create Dashboard reports in PDF or PNG format programmatically with the Reporting CLI without using the OpenSearch Dashboards and the Reporting plugin. This provides automation for reporting that can be connected to email workflows.
+
+If you want to download a CSV file, you need to have the Reporting plugin installed.
+{: .note }
+
+For any Dashboard view you can request a report in PNG, or PDF format that is sent to an email address. This can be useful to send to multiple email recipients with an email alias. The Dashboard Discover view can provide a CSV report.
+
+With the Reporting CLI, you can specify options for your report in the command line. The report is sent to an email address as a PDF attachment by default. You can also request a PNG image or a CSV file with the `--formats` argument, or embed the image in the email body instead of attaching it.
+
+You can download the report to the directory that you run the Reporting CLI, or you can get the report emailed by specifying SES or SMTP for the email transport option.
+
+You can connect to OpenSearch with any of the following authentication types:
+
+- **Basic** – Basic HTTP authentication. Use `-a basic`.
+- **Cognito** – AWS Identity access with Amazon Cognito. Use `-a cognito`.
+- **SAML** – Provides authentication between an identity provider and a service provider. Use `-a saml`. OKTA provides the SAML Third-party authentication.
+- **No auth** – No authentication bypass authentication. Use `-a none`. It also defaults to No auth if `-a` flag is not specified.
+
+To learn more about Amazon Cognito, see [What is Amazon Cognito?](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html).
+
+<!--
+### Bypass authentication option
+
+The reporting CLI tool allows you to integrate it into your own workflow or environment so that you can bypass authentication or potential security issues. For example, if you use the Reporting CLI tool within an AWS Lambda instance, no security issues would occur as long as you run the Reporting plugin in OpenSearch Dashboards. In this case, you would use "No auth" to bypass the authentication process. To specify "No Auth" use `--auth none` in your request. Lambda users should test to make sure they can bypass access to Dashboards without credentials using No Auth.  
+
+To get a list of all options, see [Reporting CLI options](#reporting-cli-options).
+-->
+
+## Download and install the Reporting CLI tool
+
+You can get the Reporting CLI tool from the NPM registry or from the OpenSearch Downloads page.
+### Install the Reporting CLI from NPM
+
+To download the .tar archive and install the reporting tool with NPM, run the following command.
```

Comment:
Download will not be required when cli is published to npm. I think after publishing this tool on npm users can directly run 
```npm i @opensearch-project/opensearch-reporting-cli```
Ref: https://www.npmjs.com/package/@opensearch-project/oui

Reply:
thanks